tests/prepegMarketOrderBaseAssetAmount.ts

```diff
@@ -151,16 +163,43 @@ describe('prepeg', () => {
 			anchor.workspace.Pyth,
 			solUsd
 		);
+		const position0Before = clearingHouse.getUserAccount().positions[0];
+		console.log(position0Before.quoteAssetAmount.eq(ZERO));
 
 		const [_pctAvgSlippage, _pctMaxSlippage, _entryPrice, newPrice] =
 			calculateTradeSlippage(
-				PositionDirection.LONG,
+				direction,
 				baseAssetAmount,
 				market0,
 				'base',
 				oraclePriceData
 			);
 
+		const [
+			acquiredBaseReserve,
+			acquiredQuoteReserve,
+			acquiredQuoteAssetAmount,
+		] = calculateTradeAcquiredAmounts(
+			direction,
+			baseAssetAmount,
+			market0,
+			'base',
+			oraclePriceData
+		);
+
+		// off by 1..
+		// const acquiredQuote = _entryPrice
+		// 	.mul(baseAssetAmount.abs())
+		// 	.div(AMM_TO_QUOTE_PRECISION_RATIO)
+		// 	.div(MARK_PRICE_PRECISION);
+
+		// console.log(
+		// 	'est acquiredQuote:',
+		// 	acquiredQuote.toNumber(),
+		// 	acquiredQuoteAssetAmount.toNumber()
+		// );
+		// assert(acquiredQuoteAssetAmount.eq(acquiredQuote));
```

Comment:
whats the deal here?

Reply:
was debugging to see if the acquiredQuoteAssetAmount could be accurately derived from entryPrice, but was off by 1

deleted